aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -239,6 +241,128 @@ Tensor wrapped_scalar_tensor(
 
 } // anonymous namespace
 
+Tensor& isin_out(const Tensor& elements, const Tensor& test_elements, bool assume_unique, bool invert, Tensor& out) {
+  out.resize_as_(elements);
+  if (elements.numel() == 0) {
+    return out;
+  }
+
+  // Condition taken from numpy's implementation.
```

Comment:
Can we put a link here to keep a reference?

Reply:
Good idea- added!